MapboxNavigation/NavigationMapView.swift

```diff
@@ -958,6 +960,226 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
             SourceIdentifier.arrowSymbol,
         ].compactMap { style.source(withIdentifier: $0) }))
     }
+
+    public func showRouteDurationAnnotations(_ routes: [Route]?) {
```

Comment:
Added some documentation.

Right now the user will need to call the `showRouteDurationAnnotations` function themselves. I did it this way since I was trying to minimize the changes I was making to existing behavior and wanted to give people more fine-grained control on when the annotations show up, when they are refreshed, and when they are removed.

Not sure if that's the best way to go but it fits my "low impact" intention.